main.go

```diff
@@ -887,7 +887,7 @@ func (s *Server) injectRoute(path *bgptable.Path) error {
 // linux kernel
 // TODO: multipath support
 func (s *Server) watchBGPPath() error {
-	watcher := s.bgpServer.Watch(bgpserver.WatchBestPath())
+	watcher := s.bgpServer.Watch(bgpserver.WatchBestPath(false))
```

Comment:
With the update, the function requires a boolean to be passed in.  All of the previous functionality is enabled by default, setting that value to `true` will allow other functionality that we probably do not want to add at this time.